src/database/migrations/2021_12_31_213410_default_notification_settings.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Seat\Services\Models\GlobalSetting;
+
+class DefaultNotificationSettings extends Migration
+{
+    const SETTINGS = [
+        'kassie.calendar.notify_create_operation',
```

Comment:
even better, using an array key when key is setting `$name` and value is setting `$value`.
This way, you can init the new frequency field with default `5,15,30`.